### PR TITLE
Add "pardon" command for votekicks

### PR DESCRIPTION
**Pull request by fuzzbuck**
Allows admins to "pardon" a votekick ban, useful for when a player gets unfairly votekicked and has to wait **60 minutes**, but he can instead be pardoned and join right away.

### CODE REVIEW

File: server/src/mindustry/server/ServerControl.java [hunk 1]
```diff
@@ -659,6 +659,16 @@ private void registerCommands(){
                 err("That IP/ID is not banned!");
             }
         });
+        
+        handler.register("pardon", "<ID>", "Pardons a votekicked player by ID and allows them to join again.", arg -> {
+            PlayerInfo info = netServer.admins.getInfo(arg[0]);
```
Comment (Anuken) [inline server/src/mindustry/server/ServerControl.java]: Use `getInfoOptional`; `getInfo` will always create an info if it does not exist, and may never return null.

File: server/src/mindustry/server/ServerControl.java [hunk 2]
```diff
@@ -659,6 +659,16 @@ private void registerCommands(){
...
+            if(info != null){
+                info.lastKicked = 0;
+                info("Pardoned player: {0}", info.lastName);
+            } else{
```
Comment (Anuken) [inline server/src/mindustry/server/ServerControl.java]: extra space

Comment (Anuken) [reply server/src/mindustry/server/ServerControl.java]: I meant the space in `} else{` before the `else`.

Comment (fuzzbuck) [reply server/src/mindustry/server/ServerControl.java]: oh

File: server/src/mindustry/server/ServerControl.java [hunk 3]
```diff
@@ -666,7 +666,7 @@ private void registerCommands(){
             if(info != null){
                 info.lastKicked = 0;
                 info("Pardoned player: {0}", info.lastName);
-            } else{
+            }else{
                 err("That ID hasn't been votekicked!");
```
Comment (Anuken) [inline server/src/mindustry/server/ServerControl.java]: Might be more accurate to say "That ID can't be found" instead.

Comment (fuzzbuck) [reply server/src/mindustry/server/ServerControl.java]: oke

**Pull request merged**